### DISCUSSION TITLE
Creating text of stops in panel

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#89] [General]
Hello,

For creating my stop text in GUI elements on my panel, instead of putting text in DispLabelText, I want to add an image on and image off with the text already noted in it. Is it possible to?

If yes, how to delete the text by default ?

**Comment by larspalo** [comment]
Set TextBreakWidth to 0.

**Reply by Jean-Gabriel1982** [reply]
Many thanks !

**Reply by Jean-Gabriel1982** [reply]
To continue with custom buttons, when I open my organ file in GO, to activate the stop I have to click top left of the button .. middle or right, nothing happens .. is it possible that the stop is activated when I click wherever on the button ?

**Reply by larspalo** [reply]
Try setting MouseRadius to 0.

**Reply by larspalo** [reply]
And check that all the MouseRect values are sensible.

**Reply by Jean-Gabriel1982** [reply]
It works many thanks !

MouseRadius is 0
MouseRect left and top are 0 and
MouseRecWidth is still 190 and Height still 50. I haven't change it ...

**Reply by Psi58** [reply]
@Jean-Gabriel1982 - I had many, many questions like this when I first started adapting organs. Lars and the wider community are always very helpful, but I found the best way to get answers (and to understand the underlying structure) is to open up an existing organ that has the features you want in GOODF and/or Notepad++ and to learn from that.
Just my 2 cents - still a beginner, but learning...😉

**Reply by eturpault** [reply]
and/or [OdfEdit](https://github.com/GrandOrgue/OdfEdit) :smile:

**Reply by larspalo** [reply]
@eturpault Yes, I think that OdfEdit and GoOdf together makes for a very powerful combo to handle more or less any odf related need! But I must admit that I anyway frequently inspect the text of the odf file in a text editor just to manually verify that it's correctly written... (or not...)  :innocent:

**Reply by eturpault** [reply]
Yes GoOdf and OdfEdit are complementary, not competitors. For an expert like you Notepad++ can be good indeed, but for someone who wants to learn how an ODF is made, OdfEdit with its sections list/tree and integrated ODF syntax help is more convenient than Notepad++ I think, hence my suggestion to @Jean-Gabriel1982.

**Reply by Jean-Gabriel1982** [reply]
@Psi58 @eturpault yes of course, I had already done that..

Perhaps with the wrong sound bank, very or even too complete, but it also gave me an overview of what could be done. 

However, there are still some practical questions for which I did not yet have the answer. Thanks to Lars for answering me so quickly and efficiently… inspecting sound banks cannot answer all the questions. 

GOodf and OdfEdit are two powerful programs (even if I haven't opened the second one yet) and I think there will still be a lot of questions.. so sorry if these are stupid questions, I promise I will look into maximum answers before 😄

**Reply by eturpault** [reply]
@Jean-Gabriel1982 : if you want we can plan to have a call together in French as we did one time in the past, so I can try to help you with the knowledge I have, even if it not as large as Palo's one.

**Reply by Jean-Gabriel1982** [reply]
> @Jean-Gabriel1982 : if you want we can plan to have a call together in French as we did one time in the past, so I can try to help you with the knowledge I have, even if it not as large as Palo's one.

Dear @eturpault gladly thank you very much! We'll have to find a time to do that 🙂
